zebra/zebra_srv6.h

```diff
@@ -139,6 +139,11 @@ srv6_sid_alloc_mode2str(enum srv6_sid_alloc_mode alloc_mode)
 	}
 }
 
+struct zebra_srv6_sid_locator {
+	struct list *clients;
```

Comment:
no more `struct list` please

Reply:
Thanks for the suggestion. I replaced `struct list` with a type-safe container.